sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/FullConfigTest.java

```diff
@@ -206,8 +204,12 @@ void configures() throws Exception {
     logger.logRecordBuilder().setBody("debug log message").setSeverity(Severity.DEBUG).emit();
     logger.logRecordBuilder().setBody("info log message").setSeverity(Severity.INFO).emit();
 
-    EventLogger eventLogger = GlobalEventLoggerProvider.get().eventLoggerBuilder("test").build();
-    eventLogger.builder("namespace.test-name").put("cow", "moo").emit();
+    ((ExtendedLogger) logger)
+        .logRecordBuilder()
+        .setEventName("namespace.test-name")
+        .setSeverity(Severity.INFO)
+        .setBody(Value.of(io.opentelemetry.api.common.KeyValue.of("cow", Value.of("moo"))))
```

Comment:
We lose a default severity of `INFO` set by the event SDK.

Also, as mentioned [here](https://github.com/open-telemetry/opentelemetry-java/pull/7053/files#r1934425575), we lose the `put` overloads for adding KeyValue entries to the body.